RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -32,35 +32,37 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
     }
 
     func deleteItem(item: Item) {
-        viewController.uiWrite {
-            guard !(item as Object).invalidated, let index = items.indexOf(item) else {
+        try! items.realm?.write {
```

Comment:
Oh dear. I think this was a merge error when I updated the branch. I've updated them to `uiWrite` now.